docs/docs/external/getting-started/interop/asset-exchange.md

```diff
@@ -0,0 +1,11 @@
+---
+id: asset-exchange
+title: Testing Interoperation Scenarios for Asset Exchange
+---
```

Comment:
Can this have `sidebar_label: Asset Exchange`?

Reply:
Fixed